Pixel Game/Assets/Scripts/AnimatorExt.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// We have to make sure not to set a value for an animator parameter that does not exist
```

Comment:
This is the script to fix the animator bug when using the bow?

Reply:
Nah, this is because the Movement script sets some animator params for walking and being idle, but not all objects have such animations. So this will make it work regardless, without warnings.